man/bst-show.1

```diff
@@ -45,6 +45,7 @@ Show elements in the pipeline
         %{deps}           A list of all dependencies
         %{build-deps}     A list of build dependencies
         %{runtime-deps}   A list of runtime dependencies
+        %{artifact-cas-digest} The CAS digest of the built artifact
```

Comment:
This change belongs in the docstring of the `show()` command in `_frontend/cli.py`, and the man pages are generated from the frontend docstrings.

Updating the man pages is [documented here](https://docs.buildstream.build/master/hacking/writing_documentation.html#man-pages) (I know, its a bit of an obscure location and the contributing guide is a bit too much and comes from days when we had way too many contributors).

Also the man page generation will not work right now, this is fixed in the `tristan/sboms` branch in 0c2a92d2667f7f88938bea8779d50484ed6b6d94